README.md

```diff
@@ -34,6 +34,30 @@ npm install
 
 alternatively for production is also on [npm](https://www.npmjs.com/package/@bbc/digital-paper-edit-api)
 
+.env file should include:
```

Comment:
I don't know if this whole block is required. Perhaps just adding the empty DB parameters in the actual .env and mentioning it in the README 👍